src/js/entities-service/entities/events.js

```diff
@@ -6,6 +7,20 @@ const TYPE = 'events';
 
 const _Model = BaseModel.extend({
   type: TYPE,
+
+  getClinician() {
+    return Radio.request('entities', 'clinicians:model', this.get('_clinician'));
+  },
+  getEditor() {
```

Comment:
I think this might be a little cleaner if we do something like:
```js
  if (!this.get('_editor')) {
    return Radio.request('entities', 'clinician:model', {
      first_name: 'RoundingWell',
      last_name: '',
     });
  }
```
with a similar assumption on `clinician.getRole` this way if/when we display events elsewhere, this special case is in one place.  However this'll mean that we'll need to `_.trim` the name as `first_name + ' ' + '';` will be one too many spaces.